source/llassetgen-cmd/main.cpp

```diff
@@ -5,14 +5,68 @@
 
 #include <llassetgen/llassetgen.h>
 
-
-
 using namespace llassetgen;
 
 int main(int argc, char** argv) {
     llassetgen::init();
 
-    std::unique_ptr<DistanceTransform> dt(new DeadReckoning());
+	// creating the face
+	FT_Face face;
+	std::string font_file = "C:/Windows/Fonts/Arial.ttf"; // path to local font
+	char letter = 'A';
+	assert(FT_New_Face(freetype, font_file.c_str(), 0, &face) == 0);
```

Comment:
Beware that this code isn't executed for release executables!
I presume you want to execute this code and check the return value as assertion when in debug mode.